lib/config.js

```diff
@@ -126,7 +126,7 @@ function warnNodeDeprecations () {
 }
 
 async function showConfig () {
-  await updateBuildInfo();
+  await updateBuildInfo(true);
```

Comment:
This seems like the least important place to not try to check github, since this is only called when you run `appium --show-config`. The other place `updateBuildInfo` is called still will trigger checking github.

Reply:
in appium.js `updateBuildInfo` is called without arguments, which sets `useGithubApiFallback` to the default value `false`

Reply:
Ignore this. I just realized the direction of the condition.